packages/core/src/components/ButtonGroup/index.tsx

```diff
@@ -1,35 +1,7 @@
 import React from 'react';
-import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import useStyles, { StyleSheet } from '../../hooks/useStyles';
 
-export type Props = {
-  /** List of components to group. */
-  children: NonNullable<React.ReactNode>;
-  /** Stack the buttons vertically. */
-  stacked?: boolean;
-};
-
-/** Horizontally align `Button`s with a consistent gutter between each. */
-export class ButtonGroup extends React.Component<Props & WithStylesProps> {
-  static defaultProps = {
-    stacked: false,
-  };
-
-  render() {
-    const { cx, children, stacked, styles } = this.props;
-
-    return (
-      <div className={cx(styles.buttonGroup, stacked && styles.buttonGroup_stacked)}>
-        {React.Children.map(children, child =>
-          child ? (
-            <div className={cx(stacked ? styles.cell_stacked : styles.cell)}>{child}</div>
-          ) : null,
-        )}
-      </div>
-    );
-  }
-}
-
-export default withStyles(({ unit }) => ({
+const styleSheet: StyleSheet = ({ unit }) => ({
```

Comment:
This one inline?

Reply:
oops, yeah. this must've been before the switch to `styles.ts`. will update! good eye